src/main/java/dev/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package dev.service.operation;
+
+public interface OperationHandler {
+    Integer update(Integer prev, Integer value);
```

Comment:
In handler you should add or delete or get from storage

Reply:
\+